source/scpd/Cpp.d

```diff
@@ -89,6 +89,8 @@ extern(C++, (StdNamespace)) extern(C++, class) struct allocator (T) {}
 
 extern(C++, (StdNamespace)) extern(C++, class) struct less (T) {}
 
+extern(C++, (StdNamespace)) extern(C++, class) struct RandHasher (T, Hasher = hash!T) { }
```

Comment:
(StdNamespace) => `std`

should fix it i guess

Reply:
Hey that worked! Thanks!